recipes-support/phytec-board-info/files/phytec-board-info.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 PHYTEC Messtechnik GmbH
+
+usage="
+Usage: $(basename $0) OPTION
+
+Print information about the current hardware.
+
+One of the following options can be selected at a time:
+  -c, --compatible        Device tree compatible of the machine
+  -m, --machine           Hardware machine name
+  -n, --simno             SIM number
+  -s, --serial            Serial number
+  -v, --machine-version   Hardware machine version
+"
+
+case "$1" in
+	-c|--compatible)
+		cut -f 1 -d '' /proc/device-tree/compatible
+		;;
+	-m|--machine)
+		cat /etc/hostname
+		;;
+	-v|--machine-version)
+		cat /proc/device-tree/model | cut -d ' ' -f 2
```

Comment:
the machine version should be the hardware version or product version. At the moment the device itself do not know it.
it is ok for the first implementation.

Reply:
This is strange. Somehow and old version of this branch slipped into this. I remember having the machine version printed here. :thinking: I will change this to print the machine version...